### PR TITLE
Add Hazelcast Authentication

### PR DESCRIPTION
**Pull request by nielsm5**

### CODE REVIEW

**Review by jkosternl — APPROVED** [review]

File: console/backend/src/main/java/org/frankframework/management/web/SpringBusExceptionHandler.java [hunk 1]
```diff
@@ -62,7 +62,7 @@ public ResponseEntity<?> toResponse(Throwable cause) {
 			return messageConverter.apply(cause);
 		}
 
-		public static ManagedException parse(Throwable cause) {
+		public static @Nullable ManagedException parse(Throwable cause) {
```
[the remark this answers is not in the document]
Comment (nielsm5) [reply console/backend/src/main/java/org/frankframework/management/web/SpringBusExceptionHandler.java:65]: Will remember this for the next time, is this part of the new formatting rules?

[the remark this answers is not in the document]
Comment (tnleeuw) [reply console/backend/src/main/java/org/frankframework/management/web/SpringBusExceptionHandler.java:65]: I think it is.

**Review by tnleeuw — APPROVED** [review]

File: console/backend/src/main/java/org/frankframework/management/web/SpringBusExceptionHandler.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (nielsm5) [reply console/backend/src/main/java/org/frankframework/management/web/SpringBusExceptionHandler.java:65]: We should format the entire project soon!

[the remark this answers is not in the document]
Comment (jkosternl) [reply console/backend/src/main/java/org/frankframework/management/web/SpringBusExceptionHandler.java:65]: Or just revert the change that put the annotation after the `public static` declaration 😉

**Pull request merged**